MekHQ/resources/mekhq/resources/Campaign.properties

```diff
@@ -82,7 +82,7 @@ turnoverPersonnelKilled.text=<html>You have personnel who have left the unit or
 divorce.text=%s has divorced %s.
 
 #### Unsorted Campaign Resources
-dependentLeavesForce.text=%s is no longer traveling with the force.
+dependentLeavesForce.text=%s dependent%s have departed the force.
```

Comment:
Add a space between `dependent` and `%s`. All string grammar should be in this file and not handled within code.